mixer/pkg/runtime/controller.go

```diff
@@ -114,7 +115,7 @@ func (c *Controller) publishSnapShot() {
 	// attribute manifests are used by type inference during handler creation.
 	attributes := c.processAttributeManifests()
 
-	if cl, ok := c.eval.(VocabularyChangeListener); ok {
+	if cl, ok := c.evaluator.(VocabularyChangeListener); ok {
```

Comment:
cc: @mandarjog 

Mandar, can you please take a look at this and tell me if this is sane?

Reply:
Lgtm